src/mysql.erl

```diff
@@ -439,9 +442,6 @@ encode(Conn, Term) ->
 
 %% --- Gen_server callbacks ---
 
--include("records.hrl").
```

Comment:
These include files define records and macros used for the gen_server callbacks. There are included here together with the definition of the gen_server state record. They are not used for the public API functions above. (The gen_server behaviour part should possibly be moved to it's own module in the future.)